Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:4.2
+// swift-tools-version:4.1
```

Comment:
@ianpartridge  couldn't this just be `4.0` now?

Reply:
So on Swift 4.1 we'd build in Swift 4.0 mode?

Reply:
@ianpartridge Swift 4.0 mode isn't a thing, there's only `4` (4.0 + 4.1), `4.2` (4.2), and `5` (5.0). Generally new language modes are only for major language revs (apart from 4.2 which is an exception )